docs/docs/use-the-factory-orchestrator-api.md

```diff
@@ -0,0 +1,67 @@
+
+# Using the Factory Orchestrator client API
+
+The Factory Orchestrator service, FactoryOrchestratorService.exe, provides a robust API surface for clients to interact with test devices via .NET Standard code. You can use these APIs to author advanced task orchestration code to programmatically interact with the service outside of what the app provides.
+
+All FactoryOrchestratorClient API calls other than `Connect()` are [asynchronous](https://docs.microsoft.com/dotnet/csharp/async).
+
+The FactoryOrchestator service uses [semver](https://semver.org/) versioning. If the target device is running a different Factory Orchestrator build than one used to create the client program, you need to ensure that the program you build will run as expected by calling Client.GetServiceVersionString() and comparaing against the client API version. If there is a major version mismatch your program may not work as expected and either the client program or test device should be updated so the major versions match. You can check the version of the client API by:
+
+- Manually inspecting the properties of the FactoryOrchestratorClientLibrary.dll file used by your program
+
+    ![version number in the properties of FactoryOrchestratorClientLibrary.dll](./images/fo-version-number.png)
+
+- Programatically by the following code snippet:
+
+```C#
+using System.Reflection;
+
+...
+
+var client = new FactoryOrchestratorClient(ipAddress);
+client.GetClientVersionString();
+```
+
+If you are writing a UWP app that uses the Factory Orchestrator Client API, you should use the FactoryOrchestratorUWPClient class instead of FactoryOrchestratorClient. The FactoryOrchestratorUWPClient APIs are identical to the FactoryOrchestratorClient APIs. The UWP Client is available in FactoryOrchestratorUWPClientLibrary.dll.
+
+The complete Factory Orchestrator Client API reference is available at: `FactoryOrchestrator\Documentation\api\index.html`.
+
+## Using FactoryOrchestratorUWPClient.dll in a UWP

```

Comment:
Can you remove this section? It is also not needed any more :)

Reply:
i did peep at the api doc gen. i'll see if i can get it to build into this in a later follow up.